src/Config/RectorConfig.php

```diff
@@ -88,7 +88,7 @@ public function enableCollectors(): void
      * Defaults in sync with https://phpstan.org/config-reference#parallel-processing
      * as we run PHPStan as well
      */
-    public function parallel(int $processTimeout = 60, int $maxNumberOfProcess = 32, int $jobSize = 20): void
+    public function parallel(int $processTimeout = 120, int $maxNumberOfProcess = 32, int $jobSize = 20): void
```

Comment:
max number need to be 16, jobsize to be 15

Reply:
Let's try one thing at a time at the moment, to see how it lands